### PR TITLE
Add support for custom ViewPager.PageTransformer.

### PR DESCRIPTION
**Pull request by mario**
Please note that this PR builds on top of https://github.com/vanniktech/Emoji/pull/333.

### CODE REVIEW

**Comment by mario** [comment]
Please note that this PR adds a dependency to the APP (not the LIBRARY) so we could showcase how it works without having to write a real page transformer. As such, I don't think having it as a dependency is a burden.

File: README.md [hunk 1]
```diff
@@ -162,6 +162,32 @@ setVariantEmoji(yourClassThatImplementsVariantEmoji)
 
 If no instance or a null instance is set the [default implementation](./emoji/src/main/java/com/vanniktech/emoji/VariantEmojiManager.java) will be used.
 
+## Animattions
```
Comment (vanniktech) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -162,6 +162,32 @@ setVariantEmoji(yourClassThatImplementsVariantEmoji)
...
+
+### Custom keyboard enter and exit animations
```
Comment (vanniktech) [inline README.md]: Why is this here?

File: app/build.gradle [hunk 3]
```diff
@@ -63,6 +63,8 @@ dependencies {
   implementation rootProject.ext.supportRecyclerView
   implementation rootProject.ext.supportEmojiBundled
 
+  implementation 'com.eftimoff:android-viewpager-transformers:1.0.1@aar'
```
Comment (vanniktech) [inline app/build.gradle]: This is fine!

File: README.md (hunk 2 above)
Comment (mario) [reply README.md]: Where would you put it? It belongs to animations?

Comment (mario) [reply README.md]: Ah, it's because like I previously said - this branch builds on the previous one. It'll disappear from the PR once the other PR is merged.

File: README.md (hunk 1 above)
Comment (mario) [reply README.md]: Fixed, thanks!

File: README.md (hunk 2 above)
Comment (vanniktech) [reply README.md]: Yeah I realized that once I saw the third PR.

**Comment by mario** [comment]
Updated and rebased.

**Comment by vanniktech** [comment]
Needs a rebase.

**Comment by mario** [comment]
Done.
